Lib/fontTools/varLib/varStore.py

```diff
@@ -120,6 +120,15 @@ def __getitem__(self, varidx):
 			delta += d * s
 		return delta
 
+	def applyScalars(self, varDataIndex, deltas):
+		varData = self._varData
+		scalars = [self._getScalar(ri) for ri in
+					varData[varDataIndex].VarRegionIndex]
+		delta = 0.
+		for d,s in zip(deltas, scalars):
+			delta += d * s
+		return delta
```

Comment:
Please share this last block with the function before it in a static method `interpolateFromDeltasAndScalars`. Again, like `VariationModel` does.

Reply:
Done